Formula/p/podman-tui.rb

```diff
@@ -0,0 +1,24 @@
+class PodmanTui < Formula
+  desc "Podman Terminal User Interface"
+  homepage "https://github.com/containers/podman-tui"
+  url "https://github.com/containers/podman-tui/archive/refs/tags/v0.17.0.tar.gz"
+  sha256 "8d8c070797e68b0a0ba44d123a839b1d891c38cef63d518998938c17d4d7dc75"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
```

Comment:
```suggestion
```

Reply:
Hi @chenrui333 

Done, removed the unnecessary build dep.